src/amo/components/AddonDetail.js

```diff
@@ -100,7 +118,7 @@ export class AddonDetailBase extends React.Component {
         >
           {status !== ENABLED ?
             <button
-              disabled={!clientSupportsAddons()}
+              disabled={!this.isCompatible()}
```

Comment:
Mostly I dislike passing around data like that when there's already a method to do it. I don't think `isCompatible()` is a bad thing to call multiple times. It's a cleaner API to me, having to pass less stuff around.